sensing/pointcloud_preprocessor/src/concatenate_data/concatenate_and_time_sync_nodelet.cpp

```diff
@@ -380,6 +380,20 @@ void PointCloudConcatenateDataSynchronizerComponent::publish()
   const auto & transformed_raw_points =
     PointCloudConcatenateDataSynchronizerComponent::combineClouds(concat_cloud_ptr);
 
+  for (const auto & e : cloud_stdmap_) {
+    if (e.second != nullptr) {
+      if (debug_publisher_) {
+        const auto accumulated_time =
+          std::chrono::duration<double, std::milli>(
+            std::chrono::nanoseconds(
+              (this->get_clock()->now() - e.second->header.stamp).nanoseconds()))
+            .count();
+        debug_publisher_->publish<tier4_debug_msgs::msg::Float64Stamped>(
+          "debug/accumulated_time_ms" + e.first, accumulated_time);
```

Comment:
Thanks for your PR :+1: 
I thought the name "accumulated time" meant that the time of something is added every frame, how about delay_time_ms?

Reply:
Actually, the time is accumulating because nodes in the pipeline do not change the header time of its input point cloud, it does not show the node's own delay. What do you think @xmfcx?

Reply:
@yukkysaito You are right, it is better to name it more clearly.

Let's name it `pipeline_latency_ms` to convey the information better, what do you think?

Having the word "pipeline" in there helps us understand what kind of delay this really is.

Reply:
@xmfcx Thanks. I agree :+1: